src/include/btree.i

```diff
@@ -1191,21 +1191,22 @@ __wt_page_las_active(WT_SESSION_IMPL *session, WT_REF *ref)
 }
 
 /*
- * __wt_btree_can_evict_dirty --
+ * __wt_btree_ckpt_blocks_eviction --
```

Comment:
Reversing the logic makes sense to me, but I prefer the previous version of the name. What do you think of `__wt_btree_dirty_eviction_blocked`?

Reply:
I've reverted all that, if you prefer the previous version of the name, fbm.